dbt/include/athena/macros/materializations/snapshots/snapshot.sql

```diff
@@ -147,7 +310,7 @@
     {% if column.data_type|lower == 'boolean' %}
```

Comment:
it has being raised before, but could we avoid to do a data_type casting and use https://github.com/dbt-athena/dbt-athena/blob/main/dbt/include/athena/macros/utils/ddl_data_type.sql this doesn't follow DRY standards, and we should avoid code repetition as much we can.